packages/@azure/eventhubs/client/lib/managementClient.ts

```diff
@@ -198,6 +198,26 @@ export class ManagementClient extends LinkEntity {
     }
   }
 
+  /**
+   * Removes the AMQP management session to the Event Hub.
+   * @returns {void} void
+   */
+   remove(): void {
+    try {
+      if (this._mgmtReqResLink) {
+        const mgmtLink = this._mgmtReqResLink;
```

Comment:
What is the reason to have this temporary constant? Why not just
```
clearTimeout(this._tokenRenewalTimer as NodeJS.Timer)
this._mgmtReqResLink.remove();
this._mgmtReqResLink = undefined;
```

Reply:
No reason, I'll change it.